core/domain/platform_parameter_registry_test.py

```diff
@@ -407,3 +415,12 @@ def test_evaluate_all_parameters(self) -> None:
                 'parameter_b': False,
             }
         )
+
+    def test_all_feature_flags_are_of_bool_type(self) -> None:
+        if constants.EMULATOR_MODE:
+            return
+        feature_flags = feature_services.get_all_feature_flag_dicts()
+        for feature in feature_flags:
+            if feature['data_type'] != 'bool':
```

Comment:
Why not use self.assertEqual with msg? (Make sure to parameterize msg to include the relevant info.)

Reply:
Will do that.